2023-09-20-issue1-seed-identification-api-for-nachet-frontend.py

```diff
@@ -0,0 +1,150 @@
+import os
+import json
+
+import ailab.db as db
+import ailab.db.nachet as nachet
+from ailab.models import openai
+
+current_working_directory = os.getcwd()
+seed_data_path = current_working_directory + "/seed-data"
+prompt_path = current_working_directory + "/nachet-data/prompt"
+wanted_files_number = 1
+url_to_seed_mapping = {}
+
+database = db.connect_db()
+cursor = db.cursor(database)
+
+### Get a list of all seeds URL
+query = f"""
+    SELECT DISTINCT
+        (regexp_matches(filtered_content,
+        'href="([^"]*seeds-identification[^"]*)"', 'g'))[1] AS seeds_url
+    FROM (
+        SELECT content AS filtered_content
+        FROM html_content
+        WHERE md5hash = '1a365c39a8afd80d438ddf1bd3c9afed'
+    ) AS filtered_data
+    LIMIT {wanted_files_number};
+"""
+cursor.execute(query)
```

Comment:
Je sais pas si c'est mieux mais tu pourrais mettre ta query directement dans le execute

Reply:
C'est compliqué, mais il vaut mieux les garder séparés. Cela peut aussi aider à éviter les attaques par injection SQL (dans le cas de formulaire web par exemple), même si ce n'est pas le cas ici.

Reply:
Oooh you're right